utils/structured_output.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import os
+import matplotlib.pyplot as plt
+from utils.logger_setup import setup_logger
+
+def structure_output(df):
```

Comment:
Same with this function
Mainly it doesnt give you any important stuff that you will use afterwards. Thats why u can use Jupyter Notebook for researching.